elasticsearch/variables.tf

```diff
@@ -0,0 +1,169 @@
+variable "project" {
+  type        = string
+  description = "Project name"
+}
+
+variable "environment" {
+  type        = string
+  description = "Environment name"
+}
+
+variable "name" {
+  type        = string
+  description = "Name to use for the Elasticsearch domain"
+}
+
+variable "elasticsearch_version" {
+  type        = string
+  description = "Version of the Elasticsearch domain"
+  default     = "6.7"
+}
+
+variable "options_rest_action_multi_allow_explicit_index" {
```

Comment:
maybe you can use the `tostring` function wherever this variable might be used to force it to a string: https://www.terraform.io/docs/configuration/functions/tostring.html